test/project_types/theme/themekit_test.rb

```diff
@@ -13,29 +13,29 @@ def test_create_theme_successful
       stat = mock
 
       context.expects(:system)
-        .with(Themekit::THEMEKIT,
-              'new',
-              '--password=boop',
-              '--store=shop.myshopify.com',
-              '--name=My Theme')
+        .with([Themekit::THEMEKIT,
+               'new',
+               '--password=boop',
+               '--store=shop.myshopify.com',
+               '--name=My Theme'].join(' '))
```

Comment:
We don't really have to join the arrays to call `system`, is there a particular reason why we started doing it here?

Reply:
The `system` call only takes comma-separated strings, and `build_command` builds the command as an array. Would it be neater if `build_command` returned a string and `system` was called with comma-separated strings, as it usually is?

Reply:
I believe in this case you should be able to 'unwrap' your command array by doing something like `ctx.system(*command)` in themekit.rb, like so:

```ruby
irb(main):001:0> def func(*args)
irb(main):002:1>   p args
irb(main):003:1> end
=> :func
irb(main):004:0> func('a', 'b', 'c')
["a", "b", "c"]
=> ["a", "b", "c"]
irb(main):005:0> arr = ['a', 'b', 'c']
=> ["a", "b", "c"]
irb(main):006:0> func(arr)
[["a", "b", "c"]]
=> [["a", "b", "c"]]
irb(main):008:0> func(*arr)
["a", "b", "c"]
=> ["a", "b", "c"]
```

This way you won't need to join your command before calling into `system`, which will ultimately work better for escaping the params.